quartz/styles/callouts.scss

```diff
@@ -104,6 +128,16 @@
   height: 18px;
   flex: 0 0 18px;
   padding-top: 4px;
+
+  //Handling icons
+  background-size: 18px 18px;
+  background-position: center;
+  background-color: var(--color);
+  -webkit-mask-image: var(--callout-icon);
```

Comment:
```suggestion
```

our css engine will generate the vendor prefixes automatically, safe to remove

Reply:
That's why I was having it generated 2 times !